packages/create-redwood-app/templates/ts/web/tsconfig.json

```diff
@@ -25,8 +25,8 @@
       "types/*": ["./types/*", "../types/*"],
       "@redwoodjs/testing": ["../node_modules/@redwoodjs/testing/web"]
     },
-    "typeRoots": ["../node_modules/@types", "./node_modules/@types"],
-    "types": ["jest", "@testing-library/jest-dom"],
+    "typeRoots": ["../node_modules/@types", "./node_modules/@types", "../node_modules/@testing-library"],
```

Comment:
@Tobbe do we need `jest` and `jest-dom`?

Reply:
I'm not sure. The test project seems to work find without `jest`. But I haven't looked into it more than that. So I'll add it back just to be sure.